src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap4/IdentityHostingStartup.cshtml

```diff
@@ -60,6 +60,10 @@ namespace @(thisNamespace)
 @:
 
 @:                services.AddDefaultIdentity<@(Model.UserClass)>()
+        if (Model.UseDefaultUI)
+        {
+@:                    .AddDefaultUI(UIFramework.Bootstrap4)
```

Comment:
Is this one really needed since Bootstrap4 is the default?

Reply:
.AddDefaultUI() still needs to be present in the generated file or the correct layout never gets loaded and the rendered visuals are broken. Visual Studio's templates add this line in every case. It appears to be an oversight in the dotnet CLI scaffolding.

Reply:
Is this due to something on the VS/scaffolding side?  Because on the runtime side AddDefaultUI is called by `AddDefaultIdentity()` anyways

Reply:
@HaoK The UI in our 2.2 Razor Pages app wasn't styled correctly unless the `AddDefaultUI` method call was chained on to `AddDefaultIdentity`.

Reply:
@HaoK -- As @scottaddie mentioned, this was an issue in 2.2. Looking at it now in 3.0, it appears to no longer be an issue. Closing this PR.

Reply:
Thanks for getting to the bottom of this. 
@CamSoper Appreciate you help with this & we look forward to more contributions from your side.